python/lsst/analysis/drp/colorColorPlot.py

```diff
@@ -191,12 +191,18 @@ def colorColorPlot(self, catPlot, plotInfo):
         ysStars = catPlot.loc[stars, yCol]
         zsStars = catPlot.loc[stars, zCol]
 
-        [vminGals, vmaxGals] = np.nanpercentile(zsGalaxies, [1, 99])
-        [vminStars, vmaxStars] = np.nanpercentile(zsStars, [1, 99])
-        galPoints = ax.scatter(xsGalaxies, ysGalaxies, c=zsGalaxies, cmap=newReds, label="Galaxies",
-                               s=0.5, vmin=vminGals, vmax=vmaxGals)
-        starPoints = ax.scatter(xsStars, ysStars, c=zsStars, cmap=newBlues, label="Stars", s=0.5,
-                                vmin=vminStars, vmax=vmaxStars)
+        if len(zsGalaxies) > 0:
+            [vminGals, vmaxGals] = np.nanpercentile(zsGalaxies, [1, 99])
+            galPoints = ax.scatter(xsGalaxies, ysGalaxies, c=zsGalaxies, cmap=newReds, label="Galaxies",
+                                   s=0.5, vmin=vminGals, vmax=vmaxGals)
+        else:
+            galPoints = None
```

Comment:
As a style choice, I think a blank line would be nice here.  But not necessary of course.